tests/send-mail.test.ts

```diff
@@ -56,3 +57,73 @@ test("parse mbox", async () => {
     ]);
     expect(parsed.to).toEqual("reviewer@example.com");
 });
+
+test("test quoted printable", async () => {
+const mbox =
+    `From 566155e00ab72541ff0ac21eab84d087b0e882a5 Mon Sep 17 00:00:00 2001
+Message-Id: <pull.12345.v17.git.gitgitgadget@example.com>
+From:   =?utf-8?B?w4Z2YXIgQXJuZmrDtnLDsA==?= Bjarmason <avarab@gmail.com>
+Date: Fri Sep 21 12:34:56 2001
+Subject: [PATCH 0/3] My first Pull Request!
+Fcc: Sent
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: quoted-printable
+MIME-Version: 1.0
+To: reviewer@example.com
+Cc: Some Body <somebody@example.com>,
+ And Somebody Else <somebody@else.org>
+
+This Pull Request contains some really important changes that I would love to
+have included in git.git.
+=31=32=33=34
+2.17.0.windows.1
+`;
+
+    const parsed = await parseMBox(mbox);
+    const body = PublicInboxGitHelper.mbox2markdown(parsed);
+    expect(body).toMatch(/1234/);
+});
+
+test("test base64", async () => {
+const mbox =
+    `From 566155e00ab72541ff0ac21eab84d087b0e882a5 Mon Sep 17 00:00:00 2001
+Message-Id: <pull.12345.v17.git.gitgitgadget@example.com>
+From:   =?utf-8?B?w4Z2YXIgQXJuZmrDtnLDsA==?= Bjarmason <avarab@gmail.com>
+Date: Fri Sep 21 12:34:56 2001
+Subject: [PATCH 0/3] My first Pull Request!
+Fcc: Sent
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: BaSe64
+MIME-Version: 1.0
+To: reviewer@example.com
+Cc: Some Body <somebody@example.com>,
+ And Somebody Else <somebody@else.org>
+
+` + Buffer.from("Base 64 Data").toString("base64");
```

Comment:
For my curiosity: why not embed it within the backticked part via `${Buffer.from("Base 64 Data")}`?

Reply:
I am not always remembering to take full advantage of template literals.  I pushed up a change to do as you suggested and made it a little more DRY.